content/teknologi/altinnstudio/altinn-api/app-api/data-element/_index.md

```diff
@@ -0,0 +1,147 @@
+---
+title: Data elements
+description: The App API to work with the data elements associated with an instance
+toc: true
+tags: [api]
+weight: 100
+---
+
+## Overview
+
+A Data element consist of two parts; its metadata document and the actual data blob. This API work primarily with the blob while keeping the metadata document updated.
+
+**basePath**
+```http
+{org}/{appname}/instances/{instanceOwnerPartyId}/{instanceGuid}/data
+```
+
+## Get data 
+
+Endpoint for downloading the data blob.
+
+```http
+GET basePath/{dataGuid}
+Accept: application/{xml/json}
```

Comment:
does the accept header need some more explanation? It isn't clear to me what I would have to use. Do we use application/json for all attachments ?